liquibase-standard/src/main/java/liquibase/parser/AbstractFormattedChangeLogParser.java

```diff
@@ -761,7 +761,7 @@ protected static void resetSequences(StringBuilder currentSequence, StringBuilde
         currentRollbackSequence.setLength(0);
     }
 
-    protected boolean handleAdditionalLines(DatabaseChangeLog changeLog, ResourceAccessor resourceAccessor, String line)
+    protected boolean handleAdditionalLines(DatabaseChangeLog changeLog, ResourceAccessor resourceAccessor, String line, StringBuilder currentSequence)
```

Comment:
Would be good to keep and deprecate the old method signature as this may be used by some extensions.